site/src/examples/file-drop-zone/Default.tsx

```diff
@@ -0,0 +1,15 @@
+import { ReactElement } from "react";
+import {
+  FileDropZone,
+  FileDropZoneIcon,
+  FileDropZoneTrigger
+} from "@salt-ds/lab";
+
+export const Default = (): ReactElement => {
+
+  return <FileDropZone style={{width: 200}}>
```

Comment:
Super minor, but the formatting seems quite strange. Can you try changing this to a short-hand arrow function e.g.

```tsx
export const Default = (): ReactElement => (
	<div />
);
```